spec/support/database_cleaner.rb

```diff
@@ -0,0 +1,31 @@
+RSpec.configure do |config|
+  config.before(:suite) do
+    DatabaseCleaner.clean_with(:truncation)
+    Test.setup_models
+    Rails.application.load_seed
+  end
+
+  config.before(:each) do
+    DatabaseCleaner.strategy = :transaction
+  end
+
+  config.before(:each, js: true) do
+    DatabaseCleaner.strategy = :truncation
```

Comment:
Agreed.

does that mean I should wrap this in a `if Capybara.current_driver == :rack_test` rather than `js: true` ?